js/app/historyStore.js

```diff
@@ -49,6 +49,13 @@ const HistoryStore = new Lang.Class({
         'current-set': GObject.ParamSpec.object('current-set', 'current-set', 'current-set',
             GObject.ParamFlags.READABLE, SetObjectModel.SetObjectModel),
         /**
+         * Property: current-subset
+         *
+         * The model for the current subset in the history store.
+         */
+        'current-subset': GObject.ParamSpec.object('current-subset', 'current-subset', 'current-subset',
```

Comment:
> Maybe move the logic that sets it from the CourseHistoryStore up into here?

The thing is then we'd be doing the engine query to get the first subset of a set for all apps, not just those that actually need it (right now only capacitate)